src/Commands/DrupalCommands.php

```diff
@@ -64,11 +64,13 @@ public function setRuntimeConfig(ConsoleCommandEvent $event)
      * @option account-name      Admin account name.
      * @option account-password  Admin account password.
      * @option account-mail      Admin email.
+     * @option database-type     Database type.
      * @option database-host     Database host.
      * @option database-port     Database port.
-     * @option database-name     Database name.
      * @option database-user     Database username.
      * @option database-password Database password.
+     * @option database-name     Database name.
+     * @option database-url      Database url.
```

Comment:
Instead of adding an additional `database-url` option that causes the other options to be ignored, let's make sure we can construct a valid SQLite database URL from the existing options.